content/tutorials/token-authentication.textile

```diff
@@ -1493,9 +1493,9 @@ blang[swift].
       end
   ```
 
-  Now you will need to customize the client. If you have not previously had experience building a basic user interface in Xcode, please refer to the "Apple developer guide":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/BuildABasicUI.html#//apple_ref/doc/uid/TP40015214-CH5-SW1 on how to build a simple UI and also learn more about "adding @IBOutlets@ and @IBActions@":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/ConnectTheUIToCode.html#//apple_ref/doc/uid/TP40015214-CH22-SW1.
+  Now we will need to customize the client. If you have not previously had experience building a basic user interface in Xcode, please refer to the "Apple developer guide":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/BuildABasicUI.html#//apple_ref/doc/uid/TP40015214-CH5-SW1 on how to build a simple UI and also learn more about "adding @IBOutlets@ and @IBActions@":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/ConnectTheUIToCode.html#//apple_ref/doc/uid/TP40015214-CH22-SW1.
 
-  First you will need to add a login button and a textfield so that the user can provide their chosen username. Go to your @ExampleViewController@ and add a @UIButton@ from @Object library@. Name the action "loginAction".
+  Firstly we will need to add a login button and a textfield so that the user can provide their chosen username. Go to your @ExampleViewController@ and add a @UIButton@ from @Object library@. Name the action "loginAction".
```

Comment:
Remove "we will need to"? Maybe simply say 'Firstly, add a login' as a clear instruction?